app/operations/application_operation.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationOperation
+  def self.call(*args)
+    new(*args).call
```

Comment:
@ailecksandr at next stage we will implement trailblazer architecture. So operations will be )